src/Compilers/Test/Utilities/VisualBasic/CompilationTestUtils.vb

```diff
@@ -15,31 +15,31 @@ Imports Xunit
 
 Friend Module CompilationUtils
 
-    Public Function CreateCompilation(trees As IEnumerable(Of SyntaxTree),
-                                      Optional references As IEnumerable(Of MetadataReference) = Nothing,
-                                      Optional options As VisualBasicCompilationOptions = Nothing,
-                                      Optional assemblyName As String = Nothing) As VisualBasicCompilation
-        If options Is Nothing Then
-            options = TestOptions.ReleaseDll
-        End If
+    Public Function CreateCompilation(
+                                       trees As IEnumerable(Of SyntaxTree),
+                              Optional references As IEnumerable(Of MetadataReference) = Nothing,
```

Comment:
Just ignore formatting change for the moment, as they are helping me at the moment. As it brings a lot of the parameters to be on screen, rather than require a right scroll to see.